### ISSUE TITLE
Dialogue Block: add keyboard controls for changing participant

### ISSUE THREAD

**Issue by creativecoder** [Automattic/jetpack#18276]
When inserting a new dialogue block within a conversation, add keyboard controls for changing the participant from the participant list in the conversation (or adding a new participant).

**Comment by allilevine** [comment]
Blocked by https://github.com/Automattic/jetpack/pull/18646, which may resolve this issue.

**Comment by creativecoder** [comment]
It seems like this is no longer needed, since the participant/speaker is now freeform text with autocomplete.
